index.html

```diff
@@ -55,12 +55,65 @@
     </select>
   </div>
   </main>
-  <footer class="page-footer footer-light transparent">
+  <footer class="page-footer">
     <div class="row center for-mobile"> <a class="github-button" href="{{ site.githuburl }}" data-size="large">Fork</a> </div>
-    <div class="row">
-      <div class="col m6 offset-m3 center"> <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala"> <span class="footer-title">{{ site.organization }}</span> </div>
+    <div class="container">
+        <div class="row">
+            <div class="col m3 s12">
+                <h5 class="white-text fine">Site Links</h5>
+                <ul>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/docs">Documentation</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/newcomer">Newcomer Docs</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://coala.io/coc">Code of Conduct</a></li>
+                    <li><a class="grey-text text-lighten-3" href="http://blog.coala.io/">Blog</a></li>
+                </ul>
+            </div>
+            <div class="col m9 s12 center">
+              <div class="center">
+                <h3 class="fine white-text">
+                  <span>
+                    <img class="footer-logo" src="https://api.coala.io/en/latest/_static/images/coala_logo.svg" alt="coala">
+                  </span>
+                  <span>{{ site.organization }}</span>
+                </h3>
+              </div>
+              <div class="center footer-icons">
+                <span class="icon">
+                  <a href="https://twitter.com/coala_io" target="_blank" class="btn center">
```

Comment:
use variables in jekyll _config.yml for all of these URLs, so that the website is easier to customise by another org wanting to adopt it.

Try to use similar variable names as https://github.com/octopress/social , e.g.

```yml
twitter:
  username: coala_io
```

You can create them adhoc, and they will be usable. e.g. `gitter: room: coala/coala/gsoc` will be available as `{{ site.gitter.room }}`

Also if the octopress jekyll social plugin rendering is useful, add the plugin, but I suspect you prefer to render using ng

Reply:
>  if the octopress jekyll social plugin rendering is useful, add the plugin

We're just showing the twitter and github URLs so I guess it will be OK, if we define them. As installing the plugin, will also require the same lines which will added if we don't install it and plugin is useful in that case if we're building a complex web application, but we're not. So, IMO it will good to define the variables w/o installing the plugin.

> prefer to render using ng

Yeah! It can be done using `ng` but won't it be good if we common usable varibales in one file, instead of splitting them and mentioning it another file? `_config.yml` is OK to have all CONSTANT variable at one place. As they're not getting changes. Yeah, if the variables values would have been dynamic, i would then prefer `app.js`, the right place to keep all dynamic variables.